cmd/utils/nodecmd/chaincmd.go

```diff
@@ -173,6 +175,12 @@ func initGenesis(ctx *cli.Context) error {
 			logger.Crit("Failed to write governance items", "err", err)
 		}
 
+		// Write the live pruning flag to database
+		if livePruning {
+			chainDB.WritePruningEnabled()
+			logger.Info("Enabling live pruning")
+		}
```

Comment:
Pruning-enable store would be done at https://github.com/klaytn/klaytn/pull/1871/files#diff-de2bd0674bf71cd6e3d92cb5621577831308e4cc990a15933e5e853f3358e4e0R275-R281

Is it not duplicated?

Reply:
It is to support the first use-case

```
kcn init --state.live-pruning genesis.json
kcn 
```